### PR TITLE
Move working group pages to universal layout

### PR DESCRIPTION
**Pull request by steventux**
https://trello.com/c/oLC1enhS/133-move-formats-that-have-a-content-list-to-universal-design

### Example working group page

https://government-frontend-pr-637.herokuapp.com/government/groups/joint-vat-consultative-committee-jvcc

screenshot from 2017-12-20 17-17-27

### Sticky link:

screenshot from 2017-12-20 17-17-52

Review app component guide:
https://government-frontend-pr-XXX.herokuapp.com/component-guide

Visual regression results:
https://government-frontend-pr-XXX.surge.sh/gallery.html

### CODE REVIEW

File: app/assets/stylesheets/helpers/_contents-with-body.scss [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+  @include media(tablet) {
+    max-width: 990px;
+  }
+
+  .app-c-back-to-top {
+    margin-left: 0;
+  }
+}
+
+.contents-list-link-wrapper.govuk-sticky-element--stuck-to-window {
```
Comment (fofr) [inline app/assets/stylesheets/helpers/_contents-with-body.scss]: This ought to be a component.

File: app/assets/stylesheets/helpers/_contents-with-body.scss [hunk 2]
```diff
@@ -0,0 +1,28 @@
...
+  background-color: $panel-colour;
+  display: inline;
```
Comment (vanitabarrett) [inline app/assets/stylesheets/helpers/_contents-with-body.scss]: Are `display: inline` and `float: left` needed? I tried toggling them off and it didn't seem to make a difference.

**Comment by maxgds** [comment]
@fofr re: The ios grey bar still showing when contents is on screen on mobile

- I can't make this happen at all when scrolling to the top on browserstack or on real devices
- I can reliably reproduce it on ios only when clicking the back to top link

However, it seems to be down to the way that ios renders the div that it jumps back to, it positions it VERY slightly off screen, the skip link is designed to show when the contents are moving off screen, so it's more of an ios positioning bug than anything else. Should we try to fix it considering that? The grey bar won't obscure anything the user wants to interact with so it feels to me like it would be ok to leave it as is?

File: app/assets/stylesheets/helpers/_contents-with-body.scss (hunk 2 above)
Comment (steventux) [reply app/assets/stylesheets/helpers/_contents-with-body.scss]: @vanitabarrett you're right - they aren't necessary - will remove.

**Comment by vanitabarrett** [comment]
It looks as though the page content is full width rather than 2/3 width? 
https://government-frontend-pr-637.herokuapp.com/government/groups/joint-vat-consultative-committee-jvcc

**Pull request merged**